bokeh/models/renderers.py

```diff
@@ -52,6 +51,10 @@ class TileRenderer(Renderer):
     Specifies the level in which to render the glyph.
     """)
 
+    render_parents = Bool(default=True, help="""
```

Comment:
if Im not mistaken, its already only in the `TileRenderer`.  It is not in `Renderer`